server/config.ts

```diff
@@ -120,6 +120,7 @@ export default {
     addTestErrorEndpoint: flag('FEATURE_ADD_TEST_ERROR_ENDPOINT', false),
     showAnalytics: flag('FEATURE_SHOW_ANALYTICS', false),
   },
+  prisonsWithAnalytics: (get('PRISONS_WITH_ANALYTICS', null, notRequiredInProduction) ?? '').split(','),
```

Comment:
If you default to `''`, do you still need the `?? ''`, I wouldn't think so given environment variables should always be string or not set at all.

I wonder if that's equivalent to:

```
prisonsWithAnalytics: get('PRISONS_WITH_ANALYTICS', '', notRequiredInProduction).split(','),
```

🤔

Reply:
the default from `get` is `null` when the env var is missing

Reply:
That's very odd, by the look of `get()`, if you pass an empty string as fallback it should return it when the environment variable is not set 🤔 🤷